src/dashboard/index.js

```diff
@@ -37,6 +38,230 @@ function render() {
         }
       });
   });
+
+    document.getElementById("activity").addEventListener("click", () => {
+        const repository = path.join("file://", __dirname, "activity.html");
+        let os = require("os");
+        fetch(repository)
+            .then((response) => response.text())
+            .then((text) => {
+                document.querySelector("#content").innerHTML = text;
```

Comment:
where this content id is defined ?

Reply:
https://github.com/chef/chef-workstation-app/blob/73f488b8283855b412222aa86001e2abd7d00415/src/dashboard/index.html#L32

Pretty much similar to this - https://github.com/chef/chef-workstation-app/blob/73f488b8283855b412222aa86001e2abd7d00415/src/dashboard/index.js#L13